src/android/io/branch/BranchSDK.java

```diff
@@ -298,17 +296,7 @@ private void initSession(CallbackContext callbackContext) {
             this.deepLinkUrl = data.toString();
         }
 
-        this.sessionListener = new SessionListener(callbackContext);
-        this.instance.initSession(this.sessionListener, data, activity);
-    }
-
-    private void reInitSession() {
-        if (this.sessionListener == null) {
-            return;
-        }
-
-        this.activity = this.cordova.getActivity();
-        this.instance.reInitSession(this.activity, this.sessionListener);
+        this.instance.initSession(new SessionListener(callbackContext), data, activity);
```

Comment:
We should also be able to get rid of the private variable `sessionListener` online 56 now right? It's no longer used anywhere since it was only used to gate `reInitSession`.